Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,72 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+}
+
+.box-2 {
+    position: relative;
+    text-align: center;
+}
+
+.box-2 h1 {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    font-size: 30pt;
+    font-weight: bolder;
+    color: #ffff;
+}
+
+.box-2 img {
+    width: 100%;
+    justify-content: center;
+    filter: brightness(40%);
+}
+
+.box-2 button {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: #ffff;
+    border: 0px;
+}
+
+.box-3 {
+   display: flex;
+   flex-wrap: wrap;
+    background-color: #F5CDB6;
+    height: 250px;
+}
+
+
+
+button {
+    position: absolute;
+    top: 80%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.box-3 .links {
+    display: flex;
+    width: 50%;
+    padding:0px 50px;
+}
+
+.box-3 .links img{
+    height: 50px;
+    margin: auto;
+    padding: 0px 10px;
+    }
+
+    .box-3 .links p{
+        margin: auto;
+        }
+    
```

Comment:
Really nice CSS looks like you've not repeated many properties and you are using classes and id's effectively great job.